debian/postinst

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+g_s_helper_user=g-s-helper
```

Comment:
I explained that in the Phab task. I am thinking of using the same user for other g-s privileged tasks like downloading the appstream.xml files for categorizing apps. Besides, this is Flatpak related but I wouldn't say it's Flatpak specific as gnome-software is the one who manages the external apps (e.g. calling the build; marking the headless apps as not installed if the runtime extension is missing; etc.)